nucliadb_node/nucliadb_node/tests/integration/test_indexing.py

```diff
@@ -270,6 +270,8 @@ async def test(
             elapsed = datetime.now() - start
             assert elapsed.seconds < 60, "too much time waiting"
 
+        await asyncio.sleep(0.5)
```

Comment:
This test is very flaky, and adding this here helps a lot. Not sure if it defeats the purpose of the test though, so please review with care.

Reply:
It doesn't change the purpose of the test, but it's kind of weird that the check after the loop fails if we are exiting the loop.

Totally unrelated, the test should be named `test_indexing_on_faulty_worker` or something more descriptive. I think we forgot to name it